sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -636,7 +638,8 @@ def test_enet_path_positive():
     # For multi output, positive parameter is not allowed
     # Test that an error is raised
     for path in [enet_path, lasso_path]:
-        assert_raises(ValueError, path, X, Y, positive=True)
+        with pytest.raises(ValueError):
```

Comment:
here as well, and the rest of them. It's safer to catch a more specific error than any `ValueError`.